app/controllers/concerns/verify_setup_completed.rb

```diff
@@ -37,8 +41,9 @@ def verify_setup_completed
     return if config_path? || setup_completed? || SiteConfig.waiting_on_first_user
 
     link = helpers.link_to("the configuration page", admin_config_path, "data-no-instant" => true)
+    missing_configs_text = missing_configs.map { |c| c.to_s.tr("_", " ") }.join(", ")
```

Comment:
We can maybe address @rhymes concern here (but this would require making the other suggested change, otherwise we check the mandatory configs too many times):

```ruby
display_missing = missing_configs.size > 3 ? missing_configs.first(3) + ["others"] : missing_configs
missing_configs_text = display_missing.map { |c| c.to_s.tr("_", " ") }.to_sentence
```

This would generate the following texts:

1 element: "community description"
2 elements: "community description and community action"
3 elements: "community description, community action, and tagline"
More than 3 elements: "community description, community action, tagline, and others"